autoendpoint/src/extractors/subscription.rs

```diff
@@ -294,6 +304,22 @@ fn validate_vapid_jwt(
         },
     };
 
+    // Dump the claims.
+    let claims = vapid.token.split('.').collect::<Vec<&str>>();
+    if let Some(claims_str) = claims.get(1) {
+        use base64::Engine;
+        info!(
+            "Vapid";
+            "sub" => &token_data.claims.sub,
+            "claims" => String::from_utf8(
+                base64::engine::general_purpose::URL_SAFE_NO_PAD
+                    .decode(claims_str)
+                    .unwrap_or_default()
+            )
+            .unwrap_or("UNKNOWN".to_owned())
```

Comment:
Technically, it's possible for the decode to fail due to `base64` pendency, but if that were the case, we probably can discard the content anyway.